documentation/modules/deploying/proc-deploy-setup-external-clients.adoc

```diff
@@ -1,50 +1,56 @@
+
+At least one listener supporting the desired authentication must be configured for the `KafkaUser`.
+
+To ensure compatibility between Kafka brokers and clients, configuration of authentication and authorization mechanisms must align:
 
-When you configure the `KafkaUser` authentication and authorization mechanisms, ensure they match the equivalent Kafka configuration:
+* `Kafka.spec.kafka.listeners[*].authentication` must match `KafkaUser.spec.authentication`
+* `Kafka.spec.kafka.authorization` must match `KafkaUser.spec.authorization`
 
-* `KafkaUser.spec.authentication` matches `Kafka.spec.kafka.listeners[*].authentication`
-* `KafkaUser.spec.authorization` matches `Kafka.spec.kafka.authorization`
+Listeners can be configured for mutual `TLS`, `SCRAM-SHA-512`, or `OAuth` authentication. 
+While mTLS always uses encryption, it's also recommended when using SCRAM-SHA-512 and OAuth 2.0 authentication.
 
-You should have at least one listener supporting the authentication you want to use for the `KafkaUser`.
+Authorization options for Kafka brokers include `simple`, `OAuth`, `OPA`, or `custom`. 
+When enabled, authorization is applied to all enabled listeners.
+
+NOTE: Authentication between Kafka users and Kafka brokers relies on matching settings.
+For example, mTLS authentication for a user is only possible if it's also enabled in the Kafka configuration.
 
-NOTE: Authentication between Kafka users and Kafka brokers depends on the authentication settings for each.
-For example, it is not possible to authenticate a user with mTLS if it is not also enabled in the Kafka configuration.
+.Automation and certificate management
 
 Strimzi operators automate the configuration process and create the certificates required for authentication:
 
-* The Cluster Operator creates the listeners and sets up the cluster and client certificate authority (CA) certificates to enable authentication with the Kafka cluster.
+* The Cluster Operator creates the listeners and sets up the cluster and client certificate authority (CA) certificates to enable authentication within the Kafka cluster.
 * The User Operator creates the user representing the client and the security credentials used for client authentication, based on the chosen authentication type.
 
 You add the certificates to your client configuration.
 
-In this procedure, the CA certificates generated by the Cluster Operator are used, but you can replace them by xref:installing-your-own-ca-certificates-str[installing your own certificates].
-You can also configure your listener to xref:proc-installing-certs-per-listener-{context}[use a Kafka listener certificate managed by an external CA (certificate authority)].
+In this procedure, the CA certificates generated by the Cluster Operator are used.
+Alternatively, you can replace them by xref:installing-your-own-ca-certificates-str[installing your own certificates].
```

Comment:
This should make it clear that it refers to custom CAs and not custom server certificates